components/CommunityBeLeaderSection/CommunityBeLeaderSection.js

```diff
@@ -0,0 +1,57 @@
+import { Box, Button, Icon } from 'ui-kit';
+
+export default function CommunityBeLeaderSection(props) {
```

Comment:
Quasi-nitpicky, but this component name feels clumsy.

What about something like `CommunityLeaderActions` or `CommunityLeaderSection`?

Reply:
Agreed. I like `CommunityLeaderActions`.